import { detailedInfoErrorExample } from '../../../utils/examplesErrors.util';

export const hasUserNotFoundExample = detailedInfoErrorExample(
  404,
  'Пользователь не найден!',
  'Not Found',
  '/user/has_user/test_1',
);
